packages/dd-trace/src/appsec/iast/path-line.js

```diff
@@ -47,6 +47,7 @@ function getFirstNonDDPathAndLineFromCallsites (callsites, externallyExcludedPat
         return {
           path: path.relative(process.cwd(), filepath),
           line: callsite.getLineNumber(),
+          column: callsite.getColumnNumber(),
```

Comment:
Could we add a test to be sure that we are not sending the column to the backend? And we are not going to send it in the future.

Reply:
added!